lighthouse-core/report/html/report-styles.css

```diff
@@ -728,6 +731,7 @@
 
 .lh-report {
   background-color: #fff;
+  min-width: var(--report-min-width);
```

Comment:
this used to be only `@media screen` do we still want to keep that or doesn't matter?

Reply:
Hm, I don't think it really matters. I moved it here because I felt like it was always true for every screen not just `max-width: 964px` but I'm not sure it matters?